OmniKit/PumpManager/PodCommsSession.swift

```diff
@@ -464,6 +465,7 @@ public class PodCommsSession {
                 podState.unfinalizedBolus = UnfinalizedDose(bolusAmount: units, startTime: Date(), scheduledCertainty: .uncertain)
                 return DeliveryCommandResult.uncertainFailure(error: podCommsError)
             }
+            podState.updateFromStatusResponse(status)
```

Comment:
getStatus() already does an `updateFromStatusResponse`

Reply:
The whole point of this PR is avoid having to do get status commands before each bolus command, so in nearly all cases there won't be any getStatus()! So if the updateFromStatusResponse() is not done in bolus() using the bolus command response, Loop will end up with yellow and red loops from not getting any device updates.